### ISSUE TITLE
Creation of webserver using Apache

### ISSUE THREAD

**Issue by AshwiniCUB** [AshwiniCUB/final-project-aesd#10]
Adding an Apache based webserver to buildroot image. It will be used to display the temperature, humidity and pressure data collected by the sensor.  It is optional part of the project. In case, ESP8266 web server doesn't work out then Apache server will be implemented.

DOD:

- Add necessary packages for Apache and for php scripting language
- Build the new image with the html or php file
- Make a web server running successfully and displaying all sensor values

Blockers: None

**Comment by AshwiniCUB** [comment]
To ensure the reliable presentation of temperature, humidity & pressure data, I have implemented a web server utilizing Apache. 

1. Added necessary packages for Apache, PHP with startup script.
2. Added the sensordata.php file at rootfs_overlay/usr/htdocs to be displayed while accessing the IP address.
3. After buildin & flashing image into the RPi4, I able to see the sensor data at the IP address: for my device  - it is 10.0.0.19/sensordata.php

Commit ID link - [link](https://github.com/cu-ecen-aeld/final-project-AshwiniCUB/commit/80a2f35d33a03d994f83b7f0191a893bbae0037c)



![Image](https://github.com/AshwiniCUB/final-project-aesd/assets/123501535/5031b381-4c82-4350-b965-0dccae46e513)
